src/MockBot.Api/Controllers/ChatController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MockBot.Api.Interfaces;
+using MockBot.Api.Models;
+
+namespace MockBot.Api.Controllers;
+
+[Route("api/chats")]
```

Comment:
```suggestion
[Route("api/[controller]")]
[ApiController]
```
This would add useful API behavior to the controller such as automatic HTTP 400 responses if models are invalid. Also the [controller] part adds a neat token replacement trick, so the [controller] part will be replaced by the 'Chat' prefix in line 8, 'ChatController', so the name of the controller will be the prefix of whatever you name the class (public class {token}Controller)